paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -140,6 +140,19 @@ def print_chronos_message(desired_state):
         )
 
 
+def confirm_to_continue(cluster_service_instances, desired_state):
+    paasta_print(f'You are about to {desired_state} the following instances:')
+    i_count = 0
+    for cluster, services_instances in cluster_service_instances:
+        for service, instances in services_instances.items():
+            for instance in instances.keys():
+                paasta_print(f'cluster = {cluster}, instance = {service}')
+                i_count += 1
+    if sys.stdin.isatty():
```

Comment:
The implicit `else` here kinda concerns me.
I think we need to default to true if we can't ask the tty. Or have a --yes option.